numpy/doc/basics.py

```diff
@@ -260,6 +260,45 @@
 value is inside an array or not.  NumPy scalars also have many of the same
 methods arrays do.
 
+Overflow Errors
+===============
+
+The fixed size of NumPy numeric types may cause overflow errors when a value
+requires more memory than available in the data type. For example ``np.power``
+evaluates ``100 * 10 ** 8`` correctly for 64-bit integers, but gives
+1874919424 (not correct) for a 32-bit integer.
```

Comment:
single backticks around `np.power` to make it a reference

Reply:
If I want `np.power` to make it a reference must I also include ``:ref:``?

Reply:
you have to use `numpy.power`, the reference lookup doesn't know about `np`. In this case you don't need the `:ref:` since the default domain is python it will look up the object as a python reference.